R/forecast.R

```diff
@@ -203,7 +219,7 @@ assert_forecast.default <- function(
   cli_abort(
     c(
       "!" = "The input needs to be a valid forecast object.",
-      "i" = "Please run `as_forecast()` first." # nolint
+      "i" = "Please convert to `forecast` object first (see {.fn as_forecast})." # nolint
```

Comment:
doesn't this need to be updated to match as_forecast_[forecast_type} etc? (and in line 221 as well)

Reply:
?as_forecast is just the man page for all as_forecast functions together, so I think it should be fine here

Reply:
Ah right got it. Do we still want a single man page? What happens when each as_forecast_type has a lot of methods?

Reply:
🤷 
Yeah good point. I kind of like the overview over everything. Should we keep it as is for know and maybe change as part of a future PR (e.g. the as_forecast_quantile.forecast_sample() one) as we see fit?